src/handlers/comment/handlers/multiplier.ts

```diff
@@ -18,13 +18,26 @@ export const multiplier = async (body: string) => {
     return;
   }
 
-  const regex = /\/multiplier @(\w+) (\d+(?:\.\d+)?)/; // /multiplier @0xcodercrane 0.5
+  const regex = /(".*?"|[^"\s]+)(?=\s*|\s*$)/g;
 
   const matches = body.match(regex);
 
+  matches?.shift();
+
   if (matches) {
-    const username = matches[1];
-    const bountyMultiplier = parseFloat(matches[2]);
+    let bountyMultiplier = 1;
+    let username = "";
+    let reason = "";
+
+    for (const part of matches) {
+      if (!isNaN(parseFloat(part))) {
+        bountyMultiplier = parseFloat(part);
+      } else if (part.startsWith("@")) {
+        username = part.substring(1);
+      } else {
+        reason = part;
```

Comment:
Shouldn't we remove single and double quotes form the `reason`?

Reply:
Updated the logic to get the `reason` without single or double quotes.